src/pxl_scripts/px/tcp_drops/data.pxl

```diff
@@ -20,7 +20,7 @@ import pxtrace
 import px
```

Comment:
Do you know why we have these files duplicated? I wonder if we should symlink them to avoid any potential divergence if they are truly supposed to be identical.

Reply:
I'm not sure, it seems like there are a couple PxL scripts which are duplicated in the `pxl_scripts/px` and `pxl_scripts/bpftrace` directories. I agree that it would make sense to symlink them or remove a duplicate. It also looks like they were added in separate commits, so it could simply be an oversight.
```sh
$ join -j 1 <(find px/ -type f -exec md5sum {} + | sort) <(find bpftrace/ -type f -exec md5sum {} + | sort) | awk '{print $2, $3}'
px/tcp_drops/manifest.yaml bpftrace/tcp_drops/manifest.yaml
px/tcp_drops/data.pxl bpftrace/tcp_drops/data.pxl
px/tcp_drops/vis.json bpftrace/tcp_drops/vis.json
px/tcp_retransmits/manifest.yaml bpftrace/tcp_retransmits/manifest.yaml
px/tcp_retransmits/vis.json bpftrace/tcp_retransmits/vis.json
px/tcp_restransmits/data.pxl bpftrace/tcp_retransmits/data.pxl
```

Reply:
Removed the duplicate `tcp_drops` script in `px` in the most recent commit. Going to open a separate PR for `tcp_restransmits`.